### PR TITLE
Ingestion: PROPOR vol 2

### PR DESCRIPTION
**Pull request by anthology-assist**

### CODE REVIEW

**Comment by mjpost** [comment]
Can you merge in `master` so the preview link is more concise?

**Comment by anthology-assist** [comment]
> Can you merge in `master` so the preview link is more concise?

Done.

File: data/xml/2024.propor.xml [hunk 1]
```diff
@@ -756,4 +756,333 @@
       <bibkey>freitas-2024-text</bibkey>
     </paper>
   </volume>
+  <volume id="2" ingest-date="2024-05-01" type="proceedings">
+    <meta>
+      <booktitle>Proceedings of the 16th International Conference on Computational Processing of Portuguese</booktitle>
+      <editor><first>Pablo</first><last>Gamallo</last></editor>
```
Comment (mjpost) [inline data/xml/2024.propor.xml:762]: We need a vol 1 / vol 2 in the book title — look at the PDF to see how to format it (add to both volumes)

Comment (mjpost) [reply data/xml/2024.propor.xml:762]: poke @anthology-assist

Comment (anthology-assist) [reply data/xml/2024.propor.xml:762]: Done.

**Review by mjpost — APPROVED** [review]

**Pull request merged**